.eslintrc

```diff
@@ -43,6 +43,7 @@
     "@typescript-eslint/no-var-requires": 0,
     "@typescript-eslint/no-explicit-any": 0,
     "@typescript-eslint/explicit-module-boundary-types": 0,
+    "@typescript-eslint/ban-types": 0,
```

Comment:
[This rule](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/ban-types.md) is kind of useful, it could avoid using Function/object/Object those unsafe types/primitives.

`typeof f === 'function'` is very typing strong checking, maybe we just use `: boolean` for that helper

Reply:
> maybe we just use : boolean for that helper

That doesn't work with type narrowing. For example:

```js
function (f: number | (() => void)) {
  if (isFunction(f)) {
    f()
  }
}
```

Will error if `isFunction` doesn't use the "is Function" assertion.